src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -1,19 +1,62 @@
-from typing import Any
+from typing import Any, Dict
 
 import numpy as np
 import pandas as pd
+from loguru import logger
+from vivarium import Artifact
 from vivarium.framework.randomness import RandomnessStream
 
+from vivarium_census_prl_synth_pop.results_processing.formatter import (
+    format_data_for_mapping,
+)
 from vivarium_census_prl_synth_pop.utilities import random_integers
 
 
-def generate_itin(
+def get_simulant_id_maps(
+    column_name: str,
+    obs_data: Dict[str, pd.DataFrame],
+    artifact: Artifact,
+    randomness: RandomnessStream,
+) -> Dict:
+    """
+    Get all maps that are indexed by `simulant_id`.
+
+    Parameters
+    ----------
+    column_name
+        Name of the column to use as an index
+    obs_data
+        Observer DataFrame with key for the observer name
+    artifact
+        A vivarium Artifact object needed by mapper
+    randomness
+        RandomnessStream to use in choosing zipcodes proportionally
+
+    Returns
+    -------
+    A dictionary of pd.Series suitable for pd.Series.map, indexed by `address_id`
+
+    """
+    if column_name != "simulant_id":
+        raise ValueError(f"Expected `simulant_id`, got `{column_name}`")
+    maps = dict()
+    output_cols_superset = [column_name, "has_ssn"]
+    formatted_obs_data = format_data_for_mapping(
+        index_name=column_name,
+        obs_results=obs_data,
+        output_columns=output_cols_superset,
+    )
+    maps.update(get_ssn_map(column_name, formatted_obs_data, randomness))
+    return maps
```

Comment:
why are you not mapping ITINs here?

Reply:
My understanding for MIC-3573 was that ITIN mapping is deferred for some observer work. I don't know if that has changed.